Makefile

```diff
@@ -82,9 +82,6 @@ race_condition_binary: build
 race_condition_run: race_condition_binary
 	./prometheus.race $(ARGUMENTS)
 
-run: binary
```

Comment:
For the sake of tutorials, I think saving a command is worth it as it makes it easier to get things up and running quickly.